nifi-mock/src/main/java/org/apache/nifi/util/MockProcessSession.java

```diff
@@ -164,7 +164,7 @@ public void commit() {
      * {@link org.apache.nifi.processor.Processor#onTrigger} commits or rolls back the
      * session
      */
-    public void clearCommited() {
+    public void clearCommitted() {
```

Comment:
Good question. To be honest, I was relying on good old travis to tell.

Seems like it passes but yes, we are renaming a public method.